plugins/roles_review.py

```diff
@@ -498,13 +507,17 @@ async def apply(member: Member, role: Role, inputs: List[Tuple[str, str]]) -> Op
 
         async def post_application() -> None:
             listing = None
+            user_id_heading = None
+            user_id = None
             thread = None
             voting = None
             try:
                 listing = await channel.send(format("{!m} ({}) requested {!M}:\n\n{}", member, member.display_name,
                         role, "\n\n".join("**{}**: {}".format(question, answer) for question, answer in inputs)),
                     allowed_mentions=AllowedMentions.none())
                 thread = await listing.create_thread(name=member.display_name)
+                user_id_heading = await thread.send("User ID:")
+                user_id = await thread.send("{}".format(member.id))
```

Comment:
These do not need to be separate messages. This whole code is a little nasty and I would prefer if the user id was tucked into the `listing` message.

Reply:
removed for now